phpstan-baseline.neon

```diff
@@ -1,5 +1,10 @@
 parameters:
 	ignoreErrors:
+		-
+			message: "#^Parameter \\#3 \\$subject of function str_replace expects array\\|string, string\\|null given\\.$#"
```

Comment:
nejde to fixnout?

Reply:
asi jo, pravděpodobně setinkový update phpstanu, moc sa mě do toho nechce hrabat

Reply:
fixnul sem https://github.com/keboola/php-table-backend-utils/pull/81/commits/57370c600d36de5408a95779a8a89729b6a4e92f